src/apprt/gtk/ui/ccw-osc-52-write-15.blp

```diff
@@ -0,0 +1,23 @@
+using Gtk 4.0;
+using Adw 1;
+translation-domain "com.mitchellh.ghostty";
+
+Adw.AlertDialog clipboard_confirmation_window {
```

Comment:
IIRC Blueprint files can contain multiple *objects*, just not multiple *templates*. I think the OSC 52 variants at least can and should be merged with their non-OSC 52 variants, and then we decide which object to use at runtime.

Reply:
Unfortunately for multiple objects in the same file it won't quite work, you get conflicting IDs on the responses:

![image](https://github.com/user-attachments/assets/0ab37733-4f36-4004-bb7f-3ee14d4a0696)

I'll have a think about folders/naming.

Reply:
Fixed.